src/components/icons/ArrowTopRightOnSquareIcon.vue

```diff
@@ -0,0 +1,12 @@
+<!-- eslint-disable max-len -->
+<template>
+  <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
+    <path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 003 8.25v10.5A2.25 2.25 0 005.25 21h10.5A2.25 2.25 0 0018 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25" />
+  </svg>
+</template>
+
+<script>
+export default {
+  name: 'ArrowTopRightOnSquareIcon'
```

Comment:
Is this from Heroicons? If so you don't need to define this component. Here is an example of how an import from Heroicons is done:
https://github.com/edge/account/blob/f92f85b1dfcb636b7b4455ca963a89d3ec4eaaf7/src/components/TopNavigation.vue#L45

Reply:
ArrowTopRightOnSquareIcon is only available in [heroicons 2.x](https://heroicons.com/) whereas Account UI is using [heroicons 1.x](https://v1.heroicons.com/). I tried upgrading the package, but realised this broke practically every other icon we use, so it seemed cleaner just to import this icon manually.